org.civicrm.hremergency/CRM/Hremergency/Upgrader/Steps/1004.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+trait CRM_Hremergency_Upgrader_Steps_1004 {
+
+    /**
+     * Upgrade CustomGroup, setting Emergency_Contacts is_reserved to Yes
+     * title was included here because of an issue with webform_civicrm
+     * @see https://www.drupal.org/project/webform_civicrm/issues/2947922
+     * @return bool
+     */
+    public function upgrade_1004() {
+        $result = civicrm_api3('CustomGroup', 'get', [
+            'sequential' => 1,
+            'return' => ["id"],
+            'name' => "Emergency_Contacts",
+            'api.CustomGroup.create' => ['id' => "\$value.id", 'is_reserved' => 1, 'title' => "Emergency Contacts"],
```

Comment:
This is a long line. I try to keep line length below 80 characters, but in some cases it can go up to 120. In this case can you move the parameters for `api.CustomGroup.create` to a separate variable and split the array across multiple lines, like:

```
$creationParams = [
  'id' => "\$value.id", 
  'is_reserved' => 1, 
  'title' => "Emergency Contacts"
];

$result = civicrm_api3('CustomGroup', 'get', [
  'sequential' => 1,
  'return' => ["id"],
  'name' => "Emergency_Contacts",
  'api.CustomGroup.create' => $creationParams,
]);
```